lib/datasource/docker/index.ts

```diff
@@ -432,6 +432,69 @@ export async function getDigest(
   return digest;
 }
 
+async function getTagsQuayRegistry(
```

Comment:
move to `./quay.ts`

Reply:
This did require some refactoring, let me know if OK with the current changes.